stories/core/InputGroup.stories.tsx

```diff
@@ -60,6 +61,9 @@ RadioGroup.args = {
   label: 'Radio group',
   message: 'Tell me how happy you are',
 };
+RadioGroup.parameters = {
+  creevey: { delay: 200 } as CreeveyStoryParams,
```

Comment:
Need to approve new image? I got this by running with `--ui` but before that I ran without this flag and tests passed...
![image](https://user-images.githubusercontent.com/36186912/129313534-17f1e20e-942c-495e-83a4-a4939ea31f3c.png)

Reply:
Oh wow, I didn't even see we had wrong image. Updated